src/App.css

```diff
@@ -0,0 +1,381 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-size: 100%;
+  font-family: 'Lato', sans-serif;
+  background-color: #ec4c8a;
+}
+
+img {
+  width: 100%;
+  height: auto;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+a:hover {
+  color: rgb(25, 25, 58);
+}
+
+::-webkit-scrollbar {
+  width: 0.7rem;
+  background-color: rgba(255, 255, 255, 0.123);
+  border-radius: 10px;
+}
+
+::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+  -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
+  border-radius: 10px;
+}
+
+::-webkit-scrollbar-thumb {
+  width: 1rem;
+  height: 1rem;
+  background-color: rgb(172, 172, 172);
+  border-radius: 10px;
+  box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.5);
+  -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.5);
+}
+
+.flex {
+  display: flex;
+}
+
+.limit {
+  padding-inline: 1rem;
+}
+
+/* App styling */
+
+.App {
+  width: 100%;
+  overflow-x: hidden;
+  background-color: #ec4c8a;
+  color: #fff;
+  min-height: 100vh;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #4369b2;
+  color: rgb(255, 255, 255);
+  padding: 0.7rem 1rem;
+  font-size: 1.1rem;
+  margin-bottom: 2rem;
+  text-transform: uppercase;
+}
+
+.more-header-text {
+  font-size: 0.5rem;
+}
+
+.header-text {
+  font-weight: 600;
+  font-size: 0.7rem;
+}
+
+.header-text:hover {
+  color: rgb(25, 25, 58);
+}
+
+.logo {
+  justify-content: center;
+  align-items: center;
+  gap: 0.15rem;
+}
+
+.logo img {
+  height: 1.2rem;
+  width: auto;
+}
+
+.header-icon {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.header svg {
+  font-size: 1.3rem;
+}
+
+.header svg:hover {
+  scale: 1.1;
+  cursor: pointer;
+}
+
+.loader {
+  margin-top: 7rem;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 20rem;
+}
+
+.search-input {
+  display: block;
+  width: 80%;
+  padding: 0.8rem 1.2rem;
+  background-color: rgb(255, 255, 255);
+  border: none;
+
+  /* border-radius: 2rem; */
+  margin-bottom: 2rem;
+  margin-inline: auto;
+  color: #000;
+  outline: #fff !important;
+}
+
+.error-message {
+  margin-top: 8rem;
+  font-size: 1.49rem;
+}
+
+.games-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
```

Comment:
- [ ] your project is an interesting one and your design is excellent however looking at the given template the home page is a two column grid, but yours is one, kindly make it look like the given design
your design :point_down: 
![Screenshot from 2023-05-25 20-40-32](https://github.com/fickryiman/Free-PC-Games/assets/66097371/3284c46a-3b66-4faa-b9fc-5e51be99e313)
given design :point_down: 
![Screenshot from 2023-04-28 11-04-25](https://github.com/fickryiman/Free-PC-Games/assets/66097371/430e269c-ab19-48b0-ac23-e6e8e4dee29e)

Reply:
changes mobile version into two template column

Reply:
![mobile-version](https://github.com/fickryiman/Free-PC-Games/assets/116701353/397b987d-4055-40a2-9f0b-5101ca2ec321)